docs/content/releases.md

```diff
@@ -6,6 +6,59 @@ doctypes: ["concept"]
 toc: true
 docs: "DOCS-616"
 ---
+## NGINX Ingress Controller 2.4.0
+
+04 Oct 2022
+
+OVERVIEW:
+
+* Updates to the TransportServer resource to support [proxy_protocol](https://github.com/nginxinc/kubernetes-ingress/tree/v2.0.1/examples/proxy-protocol) on port 443, both for regular HTTPS and TLS Passthrough traffic.
+* TransportServer now also supports defining and [external name service](https://kubernetes.io/docs/concepts/services-networking/service/#externalname), which will allow you to configure an external resolver for upstreams that use a domain name. For examples, see [externalname-services](https://github.com/nginxinc/kubernetes-ingress/tree/main/examples/custom-resources/externalname-services).
+* VirtualServer resource now supports [wildcard hostname](https://kubernetes.io/docs/concepts/services-networking/ingress/#hostname-wildcards).
+- Two new cli arguments have been added to the ingress controller.
+  - [-watch-namespace](https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/#-watch-namespace-string). This can configured by passing a comma separated list of namespaces to the `-watch-namespace` CLI argument (e.g. `-watch-namespace=ns-1,ns-2`).
+  - [-include-year](https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/#-include-year). When set, this cli argument will append the current year to the log output from the ingress controller. Example output: `I20220512 09:20:42.345457`.
+
+FEATURES:
+
+* [2986](https://github.com/nginxinc/kubernetes-ingress/pull/2986) Batch reloads at runtime.
+* [2914](https://github.com/nginxinc/kubernetes-ingress/pull/2914) Support watching multiple namespaces.
+* [2884](https://github.com/nginxinc/kubernetes-ingress/pull/2884) Include year in logs.
+* [2993](https://github.com/nginxinc/kubernetes-ingress/pull/2993) Accept proxy protocol when TLS passthrough enabled.
+* [3041](https://github.com/nginxinc/kubernetes-ingress/pull/3041) Support external name service for TansportServer.
+* [2939](https://github.com/nginxinc/kubernetes-ingress/pull/2939) Add support for wildcard hostname in VirutalServer.
+
+IMPROVEMENTS:
+
+* [2912](https://github.com/nginxinc/kubernetes-ingress/pull/2912) NIC + NAP DoS images.
```

Comment:
I don't think we want to abbreviate NIC, but probably this shouldn't be here anyway as it's just pipeline and tests. Maybe you want to add a note in the overview that we now publish NAP DoS images to our registry.